sigstore-java/src/main/java/dev/sigstore/tuf/TufClient.java

```diff
@@ -55,7 +55,21 @@ public class TufClient {
       1024; // Limit the update loop to retrieve a max of 1024 subsequent versions as expressed in
   // 5.3.3 of spec.
 
-  protected Clock clock = Clock.systemUTC();
+  private Clock clock;
+  private Verifiers.Supplier verifiers;
+
+  public TufClient(Clock clock, Verifiers.Supplier verifiers) {
+    this.clock = clock;
+    this.verifiers = verifiers;
+  }
+
+  public TufClient(Verifiers.Supplier verifiers) {
+    this(Clock.systemUTC(), verifiers);
+  }
```

Comment:
should these be public?

Reply:
They can be package private and still be available for tests?

Reply:
I'll remove public. FYI the TufClient class at the moment is really just a giant place holder for the update logic. It's going to be refactored, and the current API is not reflective in any way of what the client API will be.